crates/uv-settings/src/settings.rs

```diff
@@ -44,7 +44,7 @@ pub struct Options {
         feature = "schemars",
         schemars(
             with = "Option<Vec<String>>",
-            description = "PEP 508 style requirements, e.g. `flask==3.0.0`, or `black @ https://...`."
+            description = "PEP 508 style requirements, e.g. `fastapi==3.0.0`, or `black @ https://...`."
```

Comment:
Just wondering, why this change? Should we reference `ruff` instead?

Reply:
I dunno, just thought I would use something a little more modern, but we can do whatever. Ruff seems good yeah.

Reply:
I like to use Ruff when it's okay to have something that's not tiny.